src/main/java/ch/njol/skript/effects/EffCreateKeyedBossBar.java

```diff
@@ -0,0 +1,80 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+
+@Name("New Id-based BossBar")
+@Description("Create a new id-based bossbar")
+@RequiredPlugins("1.13+")
+@Examples({"create bar with id \"example\" and title \"title\"",
+	"set (bossbar with id \"example\"'s colour to red"})
+@Since("INSERT VERSION")
+public class EffCreateKeyedBossBar extends Effect {
+	
+	static {
+		if (Skript.classExists("org.bukkit.boss.KeyedBossBar"))
+			Skript.registerEffect(EffCreateKeyedBossBar.class, "create [id based] [boss[ ]]bar with id %string% [(and|with)] title[d] %string%");
```

Comment:
I don't think that part is needed, but it's up for debate ;D
```suggestion
			Skript.registerEffect(EffCreateKeyedBossBar.class, "create a [boss[ ]]bar with id %string% [(and|with)] title[d] %string%");
```

Reply:
We can remove it, it's not a big deal. We should make "a" optional though, as otherwise it would conflict with ExprCreateBossBar